JiraPS/Public/Remove-JiraIssue.ps1

```diff
@@ -0,0 +1,106 @@
+function Remove-JiraIssue {
+    [CmdletBinding(
+        ConfirmImpact = 'High',
+        SupportsShouldProcess
+    )]
+    param (
+        [Parameter(
+            Mandatory,
+            ValueFromPipeline,
+            Position = 0
+        )]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript(
+            {
+                if (("JiraPS.Issue" -notin $_.PSObject.TypeNames) -and (($_ -isnot [String]))) {
+                    $errorItem = [System.Management.Automation.ErrorRecord]::new(
+                        ([System.ArgumentException]"Invalid Type for Parameter"),
+                        'ParameterType.NotJiraIssue',
+                        [System.Management.Automation.ErrorCategory]::InvalidArgument,
+                        $_
+                    )
+                    $errorItem.ErrorDetails = "Wrong object type provided for Issue. Expected [JiraPS.Issue] or [String], but was $($_.GetType().Name)"
+                    $PSCmdlet.ThrowTerminatingError($errorItem)
+                    <#
+                      #ToDo:CustomClass
+                      Once we have custom classes, this check can be done with Type declaration
+                    #>
+                }
+                else {
+                    return $true
+                }
+            }
+        )]
+        [Object[]]
+        $Issue,
+
+        [Switch]
+        $IncludeSubTasks,
+
+        [PSCredential]
```

Comment:
There is a PR (#263) on `develop` which will change the definition of `-Credential`.
Please use the changed version, so that I don't have to merge again:

https://github.com/AtlassianPS/JiraPS/pull/263/files#diff-2ebb9ed7b267e9feab2af2ea05fd9422

Reply:
I think I should be able to rebase onto `develop` so this is a non-issue.

Reply:
I've rebased onto `develop`, which should automatically include #263. I AFAIK `[PSCredential]` should be analogous to `[System.Management.Automation.PSCredential]`, but I also added `[System.Management.Automation.CredentialAttribute()]` as well.